src/core/messenger/passback.ts

```diff
@@ -206,14 +206,29 @@ const init = (register: RegisterListener): void => {
 									);
 									void fastdom.mutate(() => {
 										const slotHeight = `${
-											adHeight + adLabelHeight
+											(getCurrentBreakpoint() === 'mobile'
+												? adHeight
+												: adSizes.outstreamDesktop
+														.height) + adLabelHeight
 										}px`;
 										log(
 											'commercial',
 											`Passback: setting height of passback slot to ${slotHeight}`,
 										);
 										slotElement.style.height = slotHeight;
 
+										/*The centre styling is added in here instead of where the element is created
+										because googletag removes the display style on the passbackElement */
+										passbackElement.style.display = 'flex';
+										passbackElement.style.flexDirection =
+											'column';
+										passbackElement.style.justifyContent =
+											'center';
+										passbackElement.style.alignItems =
+											'center';
+										passbackElement.style.height =
+											'calc(100% - 24px)';
```

Comment:
Is the 24px for the ad label height?

```suggestion
											`calc(100% - ${adLabelHeight}px)`;
```

Reply:
Yes Ravi. I am not sure why I missed adding the variable instead of the label height. Thank you for the suggestion.